.ci/build-n-run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+chmod +x .ci/build-n-run.sh
```

Comment:
No! I meant, you should make this file executable, so that we can track in git repository.
Remove `chmod` here.

Reply:
Run `ls -al .ci` for comparisons.